### PR TITLE
U/tobyc/deb packaging

### PR DESCRIPTION
**Pull request by tub**
Adds deb packaging to maven, adds bintray support to travis.
Release a new version with `mvn release:prepare release:clean`

### CODE REVIEW

**Comment by EvanKrall** [comment]
Looks good to me.

File: src/deb/bintray.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        "desc": "Version ${project.version}",
+        "released": "${build.timestamp}",
+        "vcs_tag": "v${project.version}",
+        "attributes": [],
+        "gpgSign": false
+    },
+
+    "files":
+        [
+        {"includePattern": "target/(.*.deb)", "uploadPattern": "$1", "matrixParams": { "deb_distribution": "lucid,trusty", "deb_component": "main", "deb_architecture": "amd64"} }
```
Comment (Rob-Johnson) [inline src/deb/bintray.json:30]: can we do xenial while we're at it?

Comment (Rob-Johnson) [inline src/deb/bintray.json:30]: actually, I guess that gets harder because you'll need to do special things for systemd. never mind.

**Comment by Rob-Johnson** [comment]
shipit

**Pull request merged**